Podfile.lock

```diff
@@ -125,7 +125,7 @@ EXTERNAL SOURCES:
     :branch: signal-master
     :git: https://github.com/WhisperSystems/JSQMessagesViewController.git
   SignalServiceKit:
-    :path: "."
```

Comment:
`Gemfile.lock` says:
```
    cocoapods (1.2.1)
```

```pod --version``` says:

```
1.2.1
```
So I appear to be on the right version.  What's your output for `pod --version`?